bundles/org.openhab.binding.mybmw/README.md

```diff
@@ -239,19 +259,19 @@ See description [Range vs Range Radius](#range-vs-range-radius) to get more info
 - Availability according to table
 - Read-only values
 
-| Channel Label                      | Channel ID                 | Type                 | conv | phev | bev_rex | bev |
-|------------------------------------|----------------------------|----------------------|------|------|---------|-----|
-| Mileage                            | mileage                    | Number:Length        |  X   |  X   |    X    |  X  |
-| Fuel Range                         | range-fuel                 | Number:Length        |  X   |  X   |    X    |     |
-| Electric Range                     | range-electric             | Number:Length        |      |  X   |    X    |  X  |
-| Hybrid Range                       | range-hybrid               | Number:Length        |      |  X   |    X    |     |
-| Battery Charge Level               | soc                        | Number:Dimensionless |      |  X   |    X    |  X  |
-| Remaining Fuel                     | remaining-fuel             | Number:Volume        |  X   |  X   |    X    |     |
-| Estimated Fuel Consumption l/100km | estimated-fuel-l-100km     | Number               |  X   |  X   |    X    |     |
-| Estimated Fuel Consumption mpg     | estimated-fuel-mpg         | Number               |  X   |  X   |    X    |     |
-| Fuel Range Radius                  | range-radius-fuel          | Number:Length        |  X   |  X   |    X    |     |
-| Electric Range Radius              | range-radius-electric      | Number:Length        |      |  X   |    X    |  X  |
-| Hybrid Range Radius                | range-radius-hybrid        | Number:Length        |      |  X   |    X    |     |
+| Channel ID                 | Type                 | conv | phev | bev_rex | bev |
```

Comment:
I am OK removing the label, but I would then make sure there is a description everywhere. E.g. `soc` may not be easily understood. I think you can move the label in this table to a description column, making it consistent everywhere.

Reply:
You're right I'll add some documentation

Reply:
done

Reply:
Much better, but maybe for consistency, capitalize the first letter in each description (or remove capital everywhere)?

Reply:
done